RELEASE/scripts/autoscend/auto_cooking.ash

```diff
@@ -260,6 +261,49 @@ boolean autoDrinkCafe(int howmany, int id)
 	return true;
 }
 
+int autoDailySpecialPrice()
+{
+	//this function provides the purchase price for the daily special item in chez snootie or gnomish microbrewery.
+
+	//check for known items whose autosell price is hidden due to being marked nodiscard
+	//check chez snootie
+	switch(daily_special())
+	{
+		case $item[banana]: return 3;
+		case $item[banana cream pie]: return 30;
+		case $item[fiery wing]: return 96;
+		case $item[forbidden sausage]: return 46;
+		case $item[ghost cucumber]: return 225;
+		case $item[laser-broiled pear]: return 465;
+	}
+	
+	//check gnonish microbrewery
+	switch(daily_special())
+	{
+		case $item[banana daiquiri]: return 30;
+		case $item[bungle in the jungle]: return 75;
+		case $item[especially salty dog]: return 462;
+	}
```

Comment:
Oh, I didn't mean literally two switch statements, I just meant a comment above each section inside the switch statement.

```suggestion
	switch(daily_special())
	{
		//chez snootie
		case $item[banana]: return 3;
		case $item[banana cream pie]: return 30;
		case $item[fiery wing]: return 96;
		case $item[forbidden sausage]: return 46;
		case $item[ghost cucumber]: return 225;
		case $item[laser-broiled pear]: return 465;
		
		//gnomish microbrewery
		case $item[banana daiquiri]: return 30;
		case $item[bungle in the jungle]: return 75;
		case $item[especially salty dog]: return 462;
	}
```